hopp/eco/finance.py

```diff
@@ -188,6 +188,10 @@ def run_capex(
         plant_config["h2_storage"]["type"] == "salt_cavern"
     ):  # salt cavern storage model includes compression
         h2_storage_capex = h2_storage_results["storage_capex"]
+    elif (
```

Comment:
This whole conditional block might be well-served by an [Enum](https://docs.python.org/3/library/enum.html) since it looks like there are a lot of options